src/app/convai-checker.component.spec.ts

```diff
@@ -205,13 +205,74 @@ class ConvaiCheckerWithAttributeInputTestComponent {
   }
 }
 
+@Component({
+  selector: 'my-comp-hide-loading-icon-after-load-setting',
+  template: `
+        <convai-checker
+           id="checker"
+           [inputId]="checkerInputId"
```

Comment:
What's the reason you're using constant attributes here instead of just hardcoding the string literal?

Reply:
No particular reason, I think both are valid. I tend to think of putting variables in data binding as more Angular-y, but I don't know what the official convention is.

Reply:
Okay, no objection.